.github/workflows/test-os.yml

```diff
@@ -33,12 +33,18 @@ jobs:
         platform:
           - windows/amd64
           - freebsd/amd64
+          - darwin/amd64
     steps:
       -
         name: Prepare
         run: |
           platform=${{ matrix.platform }}
           echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
+          if [ "${platform}" = "darwin/amd64" ]; then
+            echo "BUILD_TARGET=buildkitd" >> $GITHUB_ENV
```

Comment:
@crazy-max It [looks like](https://github.com/moby/buildkit/actions/runs/10666730469/job/29562839206?pr=5276#step:5:424) you stopped building `buildctl` this way. I have no experience with `docker/bake-action`, is it possible to ask it to build multiple targets?

Reply:
Looks good